.github/workflows/release.yml

```diff
@@ -22,6 +22,19 @@ jobs:
           ALLOW_TAG_PREFIX: "true"
           RELEASE_NAME_PREFIX: "HyP3 SDK "
 
+      -name: Get SDK version number
+        id: get_version
+        run: |
+          echo ::set-output name=version_tag::${GITHUB_REF#refs/tags/}
+
+      - name: Bump SDK version in HyP3 docs
+        uses: benc-uk/workflow-dispatch@v1.1
+        with:
+          workflow: Update HyP3 SDK version
```

Comment:
this is slightly weak as a unique identifier, but not going to worry about it

Reply:
It's a function of how that action works -- it apparently needs to be the name declared in the YAML, though it looks like we could to the workflow ID. 

https://github.com/benc-uk/workflow-dispatch#workflow

I'm not sure exactly how the workflow ID is designated (GitHub controls it) and if/when it changes, so this keeps the control in our space. I'm up to doing it either way really.